test/Array/bug8159763.js

```diff
@@ -0,0 +1,19 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+function test0() {

```

Comment:
do you want to add to one of the existing memop test cases - instead of adding a file with bug number ?